src/backend/main.ts

```diff
@@ -585,6 +585,10 @@ ipcMain.on('removeFolder', async (e, [path, folderName]) => {
     }, 2000)
   }
   return
+}
+
+ipcMain.on('removeFolder', async (e, [path, folderName]) => {
+  removeFolder(path, folderName)
```

Comment:
extracted the content of this function as a function to use it in the download manager too